portality/forms/application_forms.py

```diff
@@ -1802,6 +1804,20 @@ class FieldDefinitions:
         "name": "note",
         "group": "notes",
         "input": "textarea",
+        "disabled": "disable_edit_note_except_cur_user",
+        "contexts": {
+            "admin": {
+                "disabled" : False
+            }
+        }
+    }
+
+    # ~~->$ NoteAuthor:FormField~~
+    NOTE_AUTHOR = {
```

Comment:
Suggest we also add a placeholder text here for legacy notes where there is no author, that says something like "Original author unknown".  Placeholders can be added in `help.placeholder`, you can see an example in something like the `OA_STATEMENT_URL`